course/drillassess.php

```diff
@@ -365,9 +365,9 @@ function focusfirst() {
 			echo "<form id=\"qform\" method=\"post\" enctype=\"multipart/form-data\" action=\"$page_formAction&score=true\" onsubmit=\"doonsubmit()\">\n";
 			displayq(0,$curitemid,$seed,$doshowans,true,0);
 			if ($sa==3) {
-				echo "<input type=submit name=\"next\" value=\"Next Question\">\n";
+				echo "<button name=\"next\" value=\"Next Question\">" . _("New Question") . "</button>\n";
 			} else {
-				echo "<input type=submit name=\"check\" value=\"Check Answer\">\n";
+				echo "<button name=\"check\" value=\"Check Answer\">" . _("Check Answer") . "</button>\n";

```

Comment:
While most browsers use submit as the default type, it'd be good to add type="submit" to the button declaration for clarity.